tests/flow/fetch/headers.js

```diff
@@ -2,15 +2,15 @@
 
 const a = new Headers("'Content-Type': 'image/jpeg'"); // not correct
 const b = new Headers(['Content-Type', 'image/jpeg']); // not correct
-const c = new Headers({'Content-Type', 'image/jpeg'}); // correct
+const c = new Headers({'Content-Type': 'image/jpeg'}); // correct
```

Comment:
How was it even working!?

Reply:
IIRC, it was valid short notation, and then not being typechecked properly